client/src/api/eventstream.tsx

```diff
@@ -87,6 +88,16 @@ export const handleStreamedEvents = (
     });
   }
 
+  if (events.unculled_zones_updated) {
+    backend.get(`/map-zones/unculled`).then(
+      (result) => {
+        if (result.data) {
+          dispatch(unculledZonesUpdated(result.data));
+        }
+      }
+    );
+  }
+
```

Comment:
Check for `unculled_zones_updated` event, using `backend.get` to retrieve a list of `UnculledZoneJs` objects. Pass the data to `unculledZonesUpdated`, i.e. line 18 in `unculledZonesSlice.ts`.

Reply:
I tried using `backend.get` for the threat zones and that seems to fix the problem I described earlier, being that the threat zones don't update after a first cheat-capture. I'll probably start on a new PR for that soon, and along the way iron out similar issues with the map components.